#!/usr/bin/python3

def raise_exception_msg(message=""):
    # Raise a name exception with the provided message
    raise NameError(message)
